client/asset/eth/eth.go

```diff
@@ -501,26 +511,21 @@ func (*ExchangeWallet) PreRedeem(req *asset.PreRedeemForm) (*asset.PreRedeem, er
 	}, nil
 }
 
-// coin implements the asset.Coin interface for ETH
+// coin implements the asset.Coin interface for ETH.
 type coin struct {
-	id srveth.AmountCoinID
+	srveth.AmountCoinID
```

Comment:
I'm updating this struct in #1248, so there's no point to do this unless you disagree with those changes.

Reply:
That's fine.  I'm not going to ram this PR through, just thinking through the fund locking and nonce issue.